store/src/lmdb.rs

```diff
@@ -24,6 +24,10 @@ use lmdb_zero::LmdbResultExt;
 
 use crate::core::ser;
 
+/// number of bytes to grow the database by when needed
+pub const ALLOC_CHUNK_SIZE: usize = 134_217_728; //128 MB
```

Comment:
Seems a bit low. When Grin has full blocks, this could happen every hour or so, especially for archive peers. I'd be curious to see some metrics around how long this takes to resize. I'd also be interested to see how disruptive mdb_txn_safe::prevent_new_txns() and mdb_txn_safe::wait_no_active_txns() are (see comment on line 152). If either of those operations have a noticeable effect on performance, it'd be better to resize less often.

Reply:
Sure, can try and collect some metrics once that's implemented.

Reply:
Just another thought here, it might be debatable whether this is the right size for the chain, but it's already too large for the wallet and peer DB. I might think about making this a parameter somehow without adding too much cruft

Reply:
Good point. Maybe we should consider @antiochp's doubling proposal? Or will that eventually be too excessive?

Reply:
Seems easy enough to tune once we have live data.